tools/backups2datalad-cron

```diff
@@ -32,6 +32,12 @@ eval python -m tools.backups2datalad \
    --gh-org dandisets \
    "$*" 2>&1 | grep -v 'nothing to save, working tree clean'
 
+python -m tools.backups2datalad \
+   -l WARNING \
+   -J 5 \
+   --target "$ds" \
+   populate dandi-dandisets-dropbox "$@"
+
```

Comment:
I forgot why we had `"$*"` in above and `eval`... but may be we would need to invoke this command similar way as well.